### ISSUE TITLE
How to keep in touch with old advisor / mentor / collaborator?

### ISSUE THREAD

**Issue by nguyenthanhvuh** [dynaroars/dynaroars.github.io#4]
It is a great idea to keep in touch and reach out to those who are *close to you* during the holidays. A simple way to do this is by sending an email, for example during the Christmas - New year break, you can write something like "Dear X, just want to say hello and wish you a happy holiday season." You can also be more specific, e.g., if they are French or Argentine, say something about World Cup🙂. 

Also pay attention to how they respond to your message.  If they follow up with questions and such, then do reply.  If they just say thank you and happy holiday, then don't reply.  

In short, don't be shy and reach out to old teachers, mentors, friends, etc.
